packages/table/src/table.tsx

```diff
@@ -450,6 +450,18 @@ export class Table extends AbstractComponent<ITableProps, ITableState> {
         );
     }
 
+    public resizeRowsBasedOnColumnHeight(columnIndex: number) {
```

Comment:
Not sure what this would mean for an app - you can already select-all, double click on a column edge and have everything resize